rclpy/CMakeLists.txt

```diff
@@ -80,18 +63,15 @@ add_library(rclpy_common SHARED
   src/rclpy_common/src/common.c
   src/rclpy_common/src/handle.c
 )
-target_link_libraries(rclpy_common
-  ${PythonExtra_LIBRARIES}
+target_link_libraries(rclpy_common PUBLIC
+  pybind11::pybind11
```

Comment:
I *think* the pybind11 dependency can be `PRIVATE` here, but the rest seem to be referenced in the rclpy_common public headers.

Reply:
I think it would need to be `PUBLIC` in #675. https://github.com/ros2/rclpy/pull/675/files#diff-f3995c262a206be770fa5afb2889c8fbbbc7f1676895653d6f8f419f40c28760R64

Reply:
You're right.